site/content/events/2015-pittsburgh/propose/index.html

```diff
@@ -18,6 +18,9 @@
 This year, we are accepting all full-length (30-45 minute) proposals through our blinder system. 
 Talk proposals are accepted until <%= render(:partial => "/#{@eventhome}/_cfp_date_end") %>.
 
+We are especially interested in talks about computer ssecurity and social engineering as they relate to DevOps,
```

Comment:
I would say "computer, information, and human security and social engineering as they related to DevOps"